.github/workflows/aws-deploy.yml

```diff
@@ -0,0 +1,69 @@
+---
+name: Sample workflow to demonstrate AWS deployment including publishing data to S3 and invalidating a CloudFront distribution cache
+
+on: 
+  workflow_dispatch:
```

Comment:
@ShrimpCryptid  Just noting future work: we can add "automatic deploy to staging on tagged releases" and promoting staging to prod by manual workflow_dispatch later